mkimage.sh.in

```diff
@@ -291,9 +295,32 @@ ci20*)
     dd if="${ROOTFS}/boot/u-boot-spl.bin" of="${LOOPDEV}" obs=512 seek=1 >/dev/null 2>&1
     dd if="${ROOTFS}/boot/u-boot.img" of="${LOOPDEV}" obs=1K seek=14 >/dev/null 2>&1
     ;;
+rock64*)
+    rockchip_flash_uboot "${ROOTFS}/usr/lib/rock64-uboot" "$LOOPDEV"
+    # populate the extlinux.conf file
+    cat >"${ROOTFS}/etc/default/extlinux" <<_EOF
+TIMEOUT=3
+# Defaults to current kernel cmdline if left empty
+CMDLINE="panic=10 coherent_pool=1M console=ttyS2,1500000 root=UUID=${ROOT_UUID} rw"
+# set this to use a DEVICETREEDIR line in place of an FDT line
+USE_DEVICETREEDIR="yes"
+# relative dtb path supplied to FDT line, as long as above is unset
+DTBPATH=""
+_EOF
+    mkdir -p "${ROOTFS}/boot/extlinux"
+    run_cmd_chroot "${ROOTFS}" "/etc/kernel.d/post-install/60-extlinux"
+    # configure and enable services
+    cat >"${ROOTFS}/etc/sv/agetty-console/conf" <<_EOF
+BAUD_RATE=1500000
+TERM_NAME=linux
+_EOF
+    for _service in dhcpcd sshd agetty-console ; do
+        run_cmd_chroot "${ROOTFS}" "ln -sv /etc/sv/$_service /etc/runit/runsvdir/default/$_service"
+    done
```

Comment:
I think we should put this into it's own function, and maybe do this in a general fashion, instead of using `INSTALL` scripts in the other `-base` packages. Can you do this as a separate PR for discussion?

Reply:
hrm... wouldn't the image be unusable without these services activated?

Reply:
I mean, we should either do it `INSTALL` for all boards, or change it to be done in `mklive` for all of them. Not mixing it.

I think my personal preference would be to do the configuration of initial services for such boards in `mklive`